src/_assets/css/_header.scss

```diff
@@ -0,0 +1,217 @@
+.site-header {
```

Comment:
I suggest starting this file with something like 

```css
// Taken from http://github.com/path/to/original/file.scss. Some dartlang-specific overrides applies.
```

Another way would be to put this into `site-shared` and do the overrides, but maybe that's over-engineering things.

Reply:
I'll move this (and the _toc.scss) to the the `site-shared` repo, but first I wanted to make sure we have a working version here. As the file is an exact copy of the one in Flutter, I'd rather keep it this way (the overrides happen in main.scss).